libero-config/config--custom.js

```diff
@@ -0,0 +1,8 @@
+const config = require('./config--core');
+
+// Add your custom config properties to 'config' here.
+config.customThing = 'custom thing';
```

Comment:
This allows full access to the core config object. It's the only way I've found so far to allow arbitrary overriding of nested core properties that doesn't reset the entire property chain.